### PR TITLE
Dev 2541

### PR DESCRIPTION
**Pull request by thorstenkramm**

### CODE REVIEW

File: server/upload.go [hunk 1]
```diff
@@ -316,3 +332,29 @@ func validateUploadRequest(ur *UploadRequest) error {
 
 	return ur.Validate()
 }
+
+func validateUserRights(ur *UploadRequest, user *users.User) error {
```
Comment (domenipavec) [inline server/upload.go:336]: We shouldn't have multiple functions for checking user access to clients. There already exists 'ClientService.CheckClientsAccess` that does this, you should use it.

File: api-doc.yml [hunk 2]
```diff
@@ -1023,7 +1023,7 @@ paths:
     get:
       tags:
         - "Commands"
-      summary: "Return a short info about all multi-client commands"
+      summary: "Return a short info about all multi-client commands executed by the scheduler"
```
Comment (domenipavec) [inline api-doc.yml:1026]: I don't think the change is true, it returns all multi client commands not only those by the scheduler.

Comment (thorstenkramm) [reply api-doc.yml:1026]: It lands on `func (p *SqliteProvider) GetMultiJobSummaries()` which selects from `multi_jobs`.
The regular script execution not triggered by the schedules stores all data in the table `jobs`. 
I think we should rework this with a clear naming of the endpoints. 
Having `GET /jobs` and `GET /scheduler-jobs` would make it clearer.

Comment (domenipavec) [reply api-doc.yml:1026]: That's not true, `multi_jobs` is used any time command/script is executed on multiple clients, that includes both scheduled and manual execution.

**Review by domenipavec — DISMISSED** [review]

**Pull request merged**